admin/config.yml

```diff
@@ -43,6 +43,7 @@ collections:
   # Pages:
   - label: Pages
     name: pages
+    summary: "{{filename}} ({{title}})"
```

Comment:
This is just a style thing but I think we should either keep the title as is and include the filename as a field in the entry page or have title first and filename in parentheses.

Reply:
I had it this way because a lot of the titles are fairly long, but happy to switch it around (done)

Reply:
(imo including the filename gives a unique, concise point of reference between developers and content authors)